base/testfiles/tlb-hfloat-01.luatex.tlg

```diff
@@ -284,6 +284,6 @@ Underfull \vbox (badness 10000) has occurred while \output is active
 .\penalty 10000
 .\glue 0.0
 [1
-Caption1: 32634387
-Caption2: 35492222
+Caption1: 34426431
+Caption2: 37284266
```

Comment:
what kind of change is this?

Reply:
Because the offset changes when you set `\pdfpagesize`, etc.